UPGRADING.rst

```diff
@@ -10,6 +10,42 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4646 Rename Azul storage buckets
+=================================
+
+Operator
+~~~~~~~~
+
+After these changes are successfully merged to ``develop``, manually delete the
+old storage buckets for ``sandbox``, ``dev``, ``anvilbox``, and ``anvildev``
+as described below.
+
+After these changes are successfully merged to ``prod``, manually delete the old
+storage bucket for ``prod`` as described below.
+
+Everyone
+~~~~~~~~
+
+For each of your personal deployments, change the value of ``AZUL_S3_BUCKET`` in
+``environment.py`` to ::
+
+    "edu-ucsc-gi-{account}-storage-{AZUL_DEPLOYMENT_STAGE}.{AWS_DEFAULT_REGION}"
+
+Where ``{account}`` is the name of the AWS account hosting the deployment, e.g.,
+``"platform-hca-dev"``. As always, use the sandbox deployment's
+``environment.py`` as a model when upgrading personal deployments.
+
+After the changes are deployed to a given personal deployment, manually delete
+the old storage bucket for that deployment:
+
+1) Open the AWS console and navigate to the S3 mamangement console.
+
+2) Select the old storage bucket for your deployment. Click the "Empty" action.
+Empty the bucket and click "Exit".
```

Comment:
This renders wrong: <img width="919" alt="image" src="https://user-images.githubusercontent.com/5143256/216137540-7605834c-0a72-401c-ae51-92eea1758db8.png">

I use PyCharm's rst preview pane to check for these types of things.

Reply:
Then again, people should know how to delete buckets. Maybe we don't need this.